### DISCUSSION TITLE
Reporting Function

### DISCUSSION THREAD

**Discussion by litwinczukkarol** [rustdesk/rustdesk-server-pro#146] [Q&A]
**Hello RustDesk team,**
We are IT company interested in **Business Self-Hosting** plan of yours – only we miss one key feature in your product – **Reporting.** We would need solution similar to those shown on the screens in this email. 

![image](https://github.com/rustdesk/rustdesk/assets/150688336/e4f2ba4b-0071-4702-ae49-f43e4f35117a)

1.	In Web Console we need possibility to generate reports by supervisor / manager from his team members work. Important is that Notes section would be much wider or at least long string ( like 256 characters ) for each row.
2.	Best if Report could be exported to PDF or Excel field types
3.	Possibility to Filter entries by:
![image](https://github.com/rustdesk/rustdesk/assets/150688336/3da78be2-4401-45a0-acbd-4a00711e02f2)

![image](https://github.com/rustdesk/rustdesk/assets/150688336/33e4890f-0560-462c-bfa5-614f46639c40)

**Comment by HGrundeman** [comment]
Hi, I would like to have exact the same reporting. Did you receive an answer from the Rustdesk team? rgds Hans - VM1

**Comment by rustdesk** [comment]
Export csv does not work for you.
![image](https://github.com/rustdesk/rustdesk-server-pro/assets/71636191/edd42277-1210-460b-a742-d59afe208d1c)

**Reply by HGrundeman** [reply]
We are considering the Pro-version but until now are  using the OSS-version so I cannot check this.
